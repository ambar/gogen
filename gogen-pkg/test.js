const {mock} = require('gogen')

test('gogen', async () => {
  const {files, readFile} = await mock(__dirname, 'mylib', {})
  expect(files).toMatchSnapshot()
  expect(JSON.parse(readFile('package.json'))).toMatchSnapshot()
})
