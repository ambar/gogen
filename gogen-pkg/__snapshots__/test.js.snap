// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gogen 1`] = `
Array [
  ".gitignore",
  ".prettierrc.js",
  "LICENSE",
  "README.md",
  "index.js",
  "package.json",
  "test.js",
]
`;

exports[`gogen 2`] = `
Object {
  "description": "My awesome project",
  "license": "MIT",
  "main": "index.js",
  "name": "mylib",
  "scripts": Object {},
  "version": "0.0.0",
}
`;
